core/src/apps/cardano/layout.py

```diff
@@ -222,25 +219,20 @@ async def confirm_sending(
     network_id: int,
 ) -> None:
     if output_type == "address":
-        message = "Confirm sending"
+        title = "Sending"
     elif output_type == "change":
-        message = "Change amount"
+        title = "Change amount"
```

Comment:
with the split screen, when viewing the address, it says "Change Amount" on something that is clearly not an amount
![image](https://user-images.githubusercontent.com/596073/202718422-73be661d-7e45-4662-8361-af71db74cdb6.png)

maybe rename to "Change output"?

Reply:
fixed 2d1dbf85a2750813a44a43bac5f0b34534fa2891